vector/src/main/java/im/vector/app/features/home/room/detail/TimelineViewModel.kt

```diff
@@ -855,12 +855,12 @@ class TimelineViewModel @AssistedInject constructor(
                     R.id.invite -> state.canInvite
                     R.id.open_matrix_apps -> false // Tchap: there are no matrix apps
                     // Tchap: check if voip is enabled
-                    R.id.voice_call -> vectorFeatures.tchapIsVoipSupported() && (state.isCallOptionAvailable() || state.hasActiveElementCallWidget())
-                    // Tchap: check if voip is enabled
-                    R.id.video_call -> vectorFeatures.tchapIsVoipSupported() &&
-                            (state.isCallOptionAvailable() || state.jitsiState.confId == null || state.jitsiState.hasJoined)
+                    R.id.video_call -> vectorPreferences.developerMode()
+                    R.id.voice_call -> session.sessionParams.homeServerUrl.contains("agent.dinum.tchap.gouv.fr") &&
```

Comment:
Could it be a util functions? in case more homeserver inside this experimentation and  add new "homeserver" based feature flags

Reply:
I added the homeserver list in config feature file https://github.com/tchapgouv/tchap-android/blob/013666e79ec7d8e6098118453fdc0f1bc1c104fc/vector-config/src/tchap/res/values/config-features.xml#L8